CMakeLists.txt

```diff
@@ -353,8 +365,12 @@ endif()
 
 if(${build_as_dynamic})
     #make sure we can link as dll/so
-    add_library(aziotsharedutil_dll SHARED ./src/aziotsharedutil.def ${source_c_files} ${source_h_files})
```

Comment:
>./src/aziotsharedutil.def  [](start = 43, length = 26)

Could we please remove this .def file from the repo?

Reply:
One of the clean up items on my todo list.

---
In reply to: [184151426](https://github.com/Azure/azure-c-shared-utility/pull/152#discussion_r184151426) [](ancestors = 184151426)